portality/templates/layouts/base_dashboard.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```

Comment:
I'm not so keen to have a second full base template, as it is going to create some duplication between the other base template.  Can we just adjust the balance of what is in the one main base template, and what the new admin area template looks like.

Sorry, I realise I misunderstood when we had our meeting, I thought the base_dashboard would extend the base template, and be for use by the dashboard page.

Reply:
I thought so :) I will reorganise it to have sub-base templates for the public site vs the dashboard. Didn’t make much sense to repeat standard <head>  content

Reply:
Done in https://github.com/DOAJ/doaj/pull/2049/commits/f4a49f20659885a17b145376454d8224b8e154ae

tHANK you!